torch/csrc/autograd/functions/jit_closure.cpp

```diff
@@ -473,18 +482,15 @@ struct StageClosure {
 
     // Prepare a list of values / handles to capture
     for (auto captured_node : xstate.cur_stage_captures[stage]) {
-      if (captured_node->kind() == kSelect) {
-        auto & fn = node_map.at(captured_node->input());
-        if (captured_node->type()->kind() == TypeKind::TensorType) {
-          captured_variables.emplace_back(fn.get(), captured_node->i(kOffset), captured_node->unique());
-        } else {
-          JIT_ASSERT(captured_node->type()->kind() == TypeKind::HandleType);
-          captured_handles.emplace(fn.get(), captured_node->unique());
-        }
+      auto & fn = (input_placeholders.count(captured_node) > 0) ?
+        input_placeholders.at(captured_node)
+        : node_map.at(captured_node->node());
```

Comment:
nit: can you either put both `?` and `:` at the beginning or both at the end of a line? This syntax is quite confusing (no need to change now, just for the future)